### PR TITLE
Int 1131 address types lovs

### PR DESCRIPTION
**Pull request by LotharII**
### Jira Story

- [Add a person to a screening using address type LOVs INT-1131](https://osi-cwds.atlassian.net/browse/INT-1131)

### Purpose
Use LOVs instead of hardcoded values

### Notes for Reviewer
Should me merged with intake_api part: https://github.com/ca-cwds/intake_api/pull/215

### CODE REVIEW

File: spec/features/screening/participant/create_participant_spec.rb [hunk 1]
```diff
@@ -234,7 +234,7 @@ def build_participant_from_person_and_screening(person, screening)
         expect(page).to have_field('City', with: homer.addresses.first.city)
         expect(page).to have_field('State', with: homer.addresses.first.state)
         expect(page).to have_field('Zip', with: homer.addresses.first.zip)
-        expect(page).to have_select('Address Type', selected: homer.addresses.first.type)
+        expect(find(:css, 'select#address_type').value).to eq(homer.addresses.first.type)
```
Comment (VinhOSI) [inline spec/features/screening/participant/create_participant_spec.rb]: why did you change this?

File: spec/features/screening/participant/editing_participant_spec.rb [hunk 2]
```diff
@@ -157,7 +157,7 @@
           expect(page).to have_field('City', with: address.city)
           expect(page).to have_field('State', with: address.state)
           expect(page).to have_field('Zip', with: address.zip)
-          expect(page).to have_field('Type', with: address.type)
+          expect( find(:css, 'select#address_type').value ).to eq(address.type)
```
Comment (VinhOSI) [inline spec/features/screening/participant/editing_participant_spec.rb]: why did you change this?

Comment (LotharII) [reply spec/features/screening/participant/editing_participant_spec.rb]: because I need to check if there is correct value of selected option, in this case - code of address_type. Previous 'expect' was checking displayed text.

File: spec/features/screening/participant/create_participant_spec.rb (hunk 1 above)
Comment (LotharII) [reply spec/features/screening/participant/create_participant_spec.rb]: same as editing_participant_spec.rb
